uint32 time
float32 mag_x
float32 mag_y
float32 mag_z
